samples/modules/lvgl/demos/prj.conf

```diff
@@ -0,0 +1,17 @@
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_LOG=y
+CONFIG_SHELL=y
```

Comment:
wouldn't we want `CONFIG_INPUT=y` added here (and removed from other boards' conf?)?

Reply:
True, can be added here instead.